static/include/app/apis/generated/app.md

```diff
@@ -137,7 +137,7 @@ Return details about the requested organization.
 
 **Parameters:**
 
-- `org_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the organization to query.
+- `org_id` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The ID of the organization to query. You can retrieve this from the organization settings page.
 
 **Returns:**
 
```

Comment:
Can you change `The requested organization.` to something like
`The organization object for the requested organization containing the organizaiton's id, name, public namespace, and more.`.

I think that might be helpful

Reply:
done